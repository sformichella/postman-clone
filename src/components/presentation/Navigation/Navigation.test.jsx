import React from "react";
import { render, cleanup } from "@testing-library/react";
import Navigation from "./Navigation";

describe("Navigation component", () => {
  afterEach(() => cleanup());
  it("renders Navigation", () => {
    const { asFragment } = render(<Navigation />);
    expect(asFragment()).toMatchSnapshot();
  });
});
