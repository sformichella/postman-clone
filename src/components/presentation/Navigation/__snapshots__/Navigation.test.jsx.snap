// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation component renders Navigation 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
